packages/@aws-cdk-testing/framework-integ/test/aws-s3-deployment/test/integ.bucket-deployment.ts

```diff
@@ -144,4 +150,12 @@ listObjectsCall.expect(integ.ExpectedResult.objectLike({
   ),
 }));
 
+// Assert that there is no object keys returned from the custom resource
+const describe = integTest.assertions.awsApiCall('CloudFormation', 'describeStacks', {
+  StackName: 'test-bucket-deployments',
+});
+
+describe.assertAtPath('Stacks.0.Outputs.0.OutputKey', integ.ExpectedResult.stringLikeRegexp('customResourceData'));
+describe.assertAtPath('Stacks.0.Outputs.0.OutputValue', integ.ExpectedResult.stringLikeRegexp('Object Keys are ([0-9a-f])+\.zip'));
```

Comment:
Just want to confirm what this test is doing. In `integ.bucket-deployment-big-response.ts`, we're asserting no object keys returned by checking against regex `'Object Keys are'`, but here it seems we still have `outputObjectKeys: false` but are checking against regex `'Object Keys are ([0-9a-f])+\.zip'`.

Reply:
I added the stack output parameter as following:
```
    new CfnOutput(this, 'customResourceData', {
      value: Fn.sub('Object Keys are${keys}', {
        keys: Fn.join(',', deploymentBucket.objectKeys),
      }),
    });
  }
}
```
The reason is in the case of the `integ.bucket-deployment-big-response.ts`, `deploymentBucket.objectKeys` is expected to be an empty list, and so `Fn.join(',', deploymentBucket.objectKeys)` is expected to be empty string, and when I used describe stack API to get the output value it was returning undefined instead of an empty string, and I was not able to verify it using the available Assertions in our framework, that is why I changed the output to append some prefix string to the output, and in this case the expected result will be only the prefix, as there is no expected object keys.

In this one, the expected output is the asset name which is its name will be auto generated hash value, so I made the assertions to be like this.

Reply:
I see, I hadn't noticed this was deploying `bucket5` and not `bucket6`. In that case, we can just update the comment and should be good to go!